Sources/Errors.php

```diff
@@ -46,10 +46,7 @@ function log_error($error_message, $error_type = 'general', $file = null, $line
 
 	// are we in a loop?
 	if ($error_call > 2)
-	{
-		var_dump($backtrace);
```

Comment:
Without replacement(replace it with nothing is no replacement),
you can't remove this line of code.

Reply:
I thought this whole block was for debugging, if not I'll just revert the commit and you can deal with it then.
```
	// are we in a loop?
	if ($error_call > 2)
	{
		var_dump($backtrace);
		die('Error loop.');
	}
```

Reply:
Like i mention many many many times,
it's last >hope< when smf is in an error loop,
to get to an end AND to get the least amount of information.
In this case we can't call any smf function or do any db call or do any filesystem call.